tests/test_sshfsmount.cpp

```diff
@@ -113,7 +114,7 @@ struct SshfsMount : public mp::test::SftpServerTest
                     {
                         if (cmd == it->first)
                         {
-                            output = it->second;
+                            output = it->second + '\n';
```

Comment:
`default_cmds` are missing this treatment.

Reply:
Right, I'll investigate this. Maybe we should explicitly add `\n` in the mock returns because the real thing always answers with a newline at the end.

Reply:
Yup, either the commands should have explicit '\n', or `make_channel_read_return` should add it there.